workflow/scripts/reads_manifest.py

```
@@ -0,0 +1,32 @@
+    for ext in exts:
+        if os.path.exists(str(path) + ext):
+            return str(path) + ext
+    return "File not found"
+
+
+# remove underscores from the name field because flair does not accept them
+samples_df["sample_clean"] = samples_df["sample"].str.replace("_", "", regex=False)
+
+# get the absolute  filepath for each sample
+samples_df["sample_path"] = samples_df["sample"].apply(
+    lambda x: get_sample_path(x, snakemake.params.inputdir, exts)
```

Ruff

Undefined name `snakemake`

(F821)